dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -95,6 +95,21 @@ http.writeTimeout.desc=Amount of time to wait on a socket for each portion of th
 http.removeServerHeader=Remove server header
 http.removeServerHeader.desc=Removes server implementation information from HTTP headers and also disables the default Liberty profile welcome page.
 
+http.limitNumHeaders=Limit number of headers
+http.limitNumHeaders.desc=Use to limit the number of HTTP headers that can be present in an incoming message.If this limit is exceeded, an error is returned to the remote client.
+
+http.limitFieldSize=Limit header field size
+http.limitFieldSize.desc=Use to enforce the size limits on various HTTP fields, such as request URLs, or individual header names or values. Enforcing the size limits of these fields guards against possible Denial of Service attacks. An error is returned to the remote client if a field exceeds the allowed size.
```

Comment:
Suggestion:
Enforces the size limits on various HTTP fields, such as request URLs, or individual header names or values. Enforcing the size limits of these fields guards against possible Denial of Service attacks. An error is returned to the remote client, if a field exceeds the allowed size.